oneview_redfish_toolkit/app.py

```diff
@@ -313,23 +313,7 @@ def hp_oneview_client_exception(exception):
 
         return response
 
-    if config.auth_mode_is_conf():
-        # Loading scmb connection
-        if scmb.check_cert_exist():
-            logging.info('SCMB certs already exists testing connection...')
-        else:
-            logging.info('SCMB certs not found. Generating/getting certs....')
-            scmb.get_cert()
-            logging.info('Got certs. Testing connection...')
-        if not scmb.is_cert_working_with_scmb():
-            logging.error('Failed to connect to scmb. Aborting...')
-            exit(1)
-        scmb_thread = threading.Thread(target=scmb.listen_scmb)
-        scmb_thread.daemon = True
-        scmb_thread.start()
-    else:
-        logging.warning("Authentication mode set to session. SCMB events will "
-                        "be disabled")
+    scmb.init_event_service()
```

Comment:
Very good! :clap:
This turns clean this place of code and keeps the responsabilty of scmb logic with the scmb.py file. Nice!